index.js

```diff
@@ -55,8 +55,8 @@ import './src/filters/gamma_filter.class'; // optional image_filters
 import './src/filters/composed_filter.class'; // optional image_filters
 import './src/filters/hue_rotation.class'; // optional image_filters
 import './src/shapes/text.class'; // optional text
+import './src/mixins/itext.svg_export'; // optional svg
```

Comment:
Naming is wrong, it is a Text mixin. So it should be applied before subclassing IText